datafusion/core/src/datasource/file_format/parquet.rs

```diff
@@ -934,7 +934,7 @@ fn spawn_column_parallel_row_group_writer(
     max_buffer_size: usize,
     pool: &Arc<dyn MemoryPool>,
 ) -> Result<(Vec<ColumnWriterTask>, Vec<ColSender>)> {
-    let schema_desc = arrow_to_parquet_schema(&schema)?;
+    let schema_desc = arrow_to_parquet_schema(&schema, false)?;
```

Comment:
I found this annoying -- filed https://github.com/apache/arrow-rs/pull/6840 to make it clearer what is going on